src/nav/StreamSidebar.js

```diff
@@ -25,6 +25,8 @@ const componentStyles = StyleSheet.create({
   },
 });
 
+const SHOW_UNREAD_VIEW = true;
```

Comment:
I assume this will be false when merged to master?

Reply:
Yes!